### DISCUSSION TITLE
Profile-aware config files for dev, test & prod properties

### DISCUSSION THREAD

**Discussion by nik-christou** [quarkusio/quarkus#28456] [Community]
Hello,

Is it possible to have application-{dev/test/prod}.properties under src/main/resources ? From the documentation its clear that we can have profile aware property files for custom profile but its not clear about existing profiles that quarkus has by default.

I tried a small demo locally and it doesn't seem to work that way. Can somebody confirm this ? 
Thanks

**Comment by radcortez** [comment]
Hey @nik-christou 

Sorry for the late response. I guess I've missed this.

Yes, it works for Quarkus profiles, too (there is no distinction, actually), but you always require a main `application.properties` file (even if empty).
